atorch/examples/llama2/README.md

```diff
@@ -1,4 +1,48 @@
-## Intro
+# Llama2 Pretrain/Finetune
+
+This document presents examples of using ATorch to pretrain or finetune the HuggingFace Llama2 model, including FSDP (ZeRO3) and 3D hybrid parallelism approaches.
+
+## FSDP
+
```

Comment:
Fully Sharded Data Parallel (FSDP) is PyTorch's implementation of ZeRO3. This example uses FSDP for distributed training, and can be used with other training optimizations, such as mixed precision, gradient checkpointing, etc.